stl/inc/compare

```diff
@@ -235,21 +203,7 @@ public:
         return _Val._Value == 0;
     }
 
-#if defined(__cpp_impl_three_way_comparison) && __cpp_impl_three_way_comparison >= 201902L
     _NODISCARD friend constexpr bool operator==(const strong_ordering&, const strong_ordering&) noexcept = default;

```

Comment:
```suggestion
    _NODISCARD friend constexpr bool operator==(strong_ordering, strong_ordering) noexcept = default;
```

Has all supported compilers implemented WG21-P1946?

Reply:
They seem to have: the test passes after changing all of these, in any case.